### DISCUSSION TITLE
Ignore/scrub specific key in dictionary

### DISCUSSION THREAD

**Discussion by glen-84** [VerifyTests/Verify#807] [Q&A]
Maybe something like:

```cs
await Verify(uploadDetails)
    .ScrubMember<UploadDetails>(u => u.Fields, "Policy");
```

Where `u.Fields` is a Dictionary, and `Policy` is a key.

PS. I wonder if `SettingsTask` could be generic, so that it wouldn't be necessary to specify the generic argument to `ScrubMember`?

**Comment by SimonCropp** [comment]
does this work?
```
await Verify(uploadDetails)
    .ScrubMember("Policy");
```

> PS. I wonder if SettingsTask could be generic, so that it wouldn't be necessary to specify the generic argument to ScrubMember?

i would consider a Pull request that adds this feature

**Reply by glen-84** [reply]
That's what I have, but if I understand correctly, it would apply to any property named `Policy` within the entire object (deep), instead of just within a specific object (shallow).

**Reply by SimonCropp** [reply]
> would apply to any property named Policy within the entire object (deep

yes that is correct
